parts/k8s/kubernetesmasterresources.t

```diff
@@ -313,11 +305,10 @@
           "name": "nicLoopNode"
         },
         "dependsOn": [
-  {{if .MasterProfile.IsCustomVNET}}
```

Comment:
Before: if custom VNET depend on nsgID in master NIC / else depend on vnetID in master NIC

After: if custom VNET depend on nsgID in master NIC / else depend on vnetID + nsgID in master NIC